definitions/npm/ramda_v0.26.x/flow_v0.104.x-/ramda_v0.26.x.js

```diff
@@ -652,8 +652,11 @@ declare module ramda {
     x: E,
   ): (xs: Array<E>) => Array<E>;
 
-  declare function chain<A, B>(f: (x: A) => B[], xs: A[]): B[]
-  declare function chain<A, B>(f: (x: A) => B[]): (xs: A[]) => B[]
+  declare function chain<A, B>(f: (x: A) => B[], xs: $ReadOnlyArray<A>): B[];
```

Comment:
For consistency, I'd suggest changing `B[]` to `Array<B>` (to parallel using `$ReadOnlyArray<A>`).

Reply:
Done